src/FSCommon.h

```diff
@@ -23,9 +23,16 @@
 #endif
 
 #if defined(ARCH_ESP32)
+#if CONFIG_IDF_TARGET_ESP32S3
+#include <LittleFS.h>   //esp32s3 uses the framework's built-in LittleFS
```

Comment:
Just a note to myself, as soon as we move the main framework for legacy ESP32 beyond 3.5 we wanna use the builtin libs as well.